Jenkinsfile

```diff
@@ -105,6 +105,7 @@ pipeline {
                 checkout scm
```

Comment:
```suggestion
                 checkout scm
                MavenRun('-DskipTests=true verify')
```

Reply:
I've added `-DskipTests=true` since they have already been executed in a previous step and re-executing them would just slow down the pipeline

Reply:
Is it `verify` or `package` ?

Reply:
In the last verision I've added `-DskipTests=true package` let me know if I should use `verify` instead

Reply:
You're right @orviz, package should be enough